tests/utilities/graphqlConnectionFactory.spec.ts

```diff
@@ -1,12 +1,180 @@
-import { describe, expect, it } from "vitest";
-import { graphqlConnectionFactory } from "../../src/utilities/graphqlConnectionFactory";
+import { beforeAll, describe, expect, it } from "vitest";
+import {
+  graphqlConnectionFactory,
+  getLimit,
+  getFilterObject,
+  getSortingObject,
+  generateConnectionObject,
+} from "../../src/utilities/graphqlConnectionFactory";
+import { type CursorPaginationInput } from "../../src/types/generatedGraphQLTypes";
+import { Types } from "mongoose";
+import { nanoid } from "nanoid";
 
-describe("utilities -> graphqlConnectionFactory", () => {
+describe("utilities -> graphqlConnectionFactory -> graphqlConnectionFactory", () => {
   it(`Returns a connection object with default/pre-defined fields which
 represents a connection that has no data at all and cannot be paginated.`, () => {
     const connection = graphqlConnectionFactory<unknown>();
 
     expect(connection).toEqual({
+      edges: [],
+      pageInfo: {
+        startCursor: null,
+        endCursor: null,
+        hasNextPage: false,
+        hasPreviousPage: false,
+      },
+    });
+  });
+});
+
+describe("utilities -> graphqlConnectionFactory -> getLimit", () => {
+  it(`Should return 1 + limit if the cursor is not supplied`, () => {
+    const args: CursorPaginationInput = {
+      limit: 10,
+      direction: "FORWARD",
+    };
+
+    expect(getLimit(args)).toBe(11);
```

Comment:
Don't use hardcoded values, use the variables. Not only here, across all the places in other tests

Reply:
Got it. I am using hardcoded lengths in all other tests to manually ensure that the correct objects are fetched in correct order. Their data, though, is randomly generated every time.